lib/game.rb

```diff
@@ -12,52 +11,82 @@ def initialize(user, auto)
   end
 
   def display_board
-    message_turn + message_computer_display + message_player_display
+    message_turn
```

Comment:
Thank you for updating this. Working through the call of these methods was both fun and frustrating. I am so glad we learned so much about our print lines and about our method calling :)